components/Header.js

```diff
@@ -2,32 +2,38 @@ import styled from "styled-components";
 import Image from "next/image";
 import Link from "next/link";
 import { format } from "date-fns";
+import DefaultButton from "@/components/DefaultButton";
 
-export default function Header({ trip }) {
+export default function Header({ name, image, startDate, endDate }) {
   return (
     <StyledHeader>
       <StyledImageWrapper>
         <StyledImage
-          src={trip.image}
+          src={image}
           height={450}
           width={600}
           layout="responsive"
-          alt={trip.name}
+          alt={name}
         />
         <StyledLogo>Travel</StyledLogo>
-        <StyledTravelLogLink href="/travellog">
-          <StyledTravelLogLinkImage
-            src="/book.svg"
-            height={40}
-            width={40}
-            alt="travel log"
-          />
-        </StyledTravelLogLink>
-        <StyledName>{trip.name}</StyledName>
-        <StyledDate>
-          {format(new Date(trip.startDate), "dd/MM/yy")} -{" "}
-          {format(new Date(trip.endDate), "dd/MM/yy")}
-        </StyledDate>
+        <DefaultButton slug="travellog" icon="book" />
```

Comment:
```suggestion
        <DefaultButton url="/travellog" icon="book" />
```
You are not receiving a slug in the component, so i guess you mean the url?